ivy/functional/frontends/jax/numpy/creation.py

```diff
@@ -243,3 +243,13 @@ def compress(condition, a, *, axis=None, out=None):
         )
     arr = arr[: condition_arr.shape[0]]
     return ivy.moveaxis(arr[condition_arr], 0, axis)
+
+
+@to_ivy_arrays_and_back
+def iterable(x):
```

Comment:
Can we here assert if the given object has `__iter__` method this can be easily handled using `hasattr` function

Reply:
yeah, done!